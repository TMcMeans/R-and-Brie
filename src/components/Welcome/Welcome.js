import React from 'react';

export const Welcome = () => {
  return (
    <div className="welcome">
      <h1>Welcome, fellow cheese lover!</h1>
      <p>
        This app is designed to help you discover and favorite recipes by
        searching popular cheese types as the main ingredient. Select a cheese
        type at the top to view related recipes.
      </p>
      <p>Enjoy, and bon apetit!</p>
    </div>
  );
};
