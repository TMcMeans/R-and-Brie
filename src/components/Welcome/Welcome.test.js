import React from 'react'
import { shallow } from 'enzyme'
import { Welcome } from './Welcome'

describe('Welcome', () => {
  it('should match snapshot with all data passed in', () => {
    const wrapper = shallow(<Welcome />)
    expect(wrapper).toMatchSnapshot()
  })
})