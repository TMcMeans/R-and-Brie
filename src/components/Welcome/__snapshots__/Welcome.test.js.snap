// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match snapshot with all data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Welcome, fellow cheese lover!
        </h1>,
        <p>
          This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.
        </p>,
        <p>
          Enjoy, and bon apetit!
        </p>,
      ],
      "className": "welcome",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome, fellow cheese lover!",
        },
        "ref": null,
        "rendered": "Welcome, fellow cheese lover!",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.",
        },
        "ref": null,
        "rendered": "This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Enjoy, and bon apetit!",
        },
        "ref": null,
        "rendered": "Enjoy, and bon apetit!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Welcome, fellow cheese lover!
          </h1>,
          <p>
            This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.
          </p>,
          <p>
            Enjoy, and bon apetit!
          </p>,
        ],
        "className": "welcome",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome, fellow cheese lover!",
          },
          "ref": null,
          "rendered": "Welcome, fellow cheese lover!",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.",
          },
          "ref": null,
          "rendered": "This app is designed to help you discover and favorite recipes by searching popular cheese types as the main ingredient. Select a cheese type at the top to view related recipes.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Enjoy, and bon apetit!",
          },
          "ref": null,
          "rendered": "Enjoy, and bon apetit!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
